python/private/versions.bzl

```diff
@@ -15,32 +15,33 @@
 """The Python versions we use for the toolchains.
 """
 
-RELEASE_URL = "https://github.com/indygreg/python-build-standalone/releases/download/20211017"
-RELEASE_DATE = "20211017T1616"
+RELEASE_URL = "https://github.com/indygreg/python-build-standalone/releases/download/20220227"
+RELEASE_DATE = "20220227"
 
-# The integrity hashes can be computed with:
-# shasum -b -a 384 [downloaded file] | awk '{ print $1 }' | xxd -r -p | base64
 # buildifier: disable=unsorted-dict-items
 TOOL_VERSIONS = {
     "3.8.12": {
-        "x86_64-apple-darwin": "sha384-es0kCVBb4q5xSC09lOw83TKXtR6qdt0NeU56JtK7Y5M5V784k9MM2q8leE3QWGH6",
-        "x86_64-unknown-linux-gnu": "sha384-61BjosOUm+Uo2yDtkLcRqbSIAXIPiHKmvySGO2J/GSOHXkj5QBf4cwzrDuOWR0H2",
+        "x86_64-apple-darwin": "sha256-wmUK2QuFpdJRh5yvGHG9zgKtLGC0e3rBt2aRJ4tOto8=",
```

Comment:
Can we provide these integrity strings as 'standard' hex sha256 strings? I see that the docs say use "Subresource Integrity format" but it seems like a small usability issue that the source code integrity strings are in a different format to what Bazel spits out in logs.

Reply:
I see no reason why these couldn't become `sha256` values. It would just require updating the attribute in `repositories.bzl` I leave this decision up to @f0rmiga https://github.com/bazelbuild/rules_python/pull/618#discussion_r815470830